### DISCUSSION TITLE
Kawpow integration

### DISCUSSION THREAD

**Discussion by blackmennewstyle** [coinfoundry/miningcore#1589] [Q&A]
Hi @oliverw, i see there is an incomplete integration of `Kawpow` currently in `miningcore`. Was it intended to be integrated within the `bitcoin` family since `Ravencoin` is a fork of `BTC` or was it intended to be integrated in a whole new family?

**Comment by oliverw** [comment]
@blackmennewstyle Sorry for the delay. Been battling with Covid and then another flu since pretty much christmas. Yeah fun 😁 

It was for RVN yes, but it never quite worked out.

**Reply by blackmennewstyle** [reply]
It's all good Olivier, better late than never :)
Sorry to hear about all the infections you have been through since `Christmas` and glad to hear you are feeling better now :muscle:

**Reply by oliverw** [reply]
> It's all good Olivier, better late than never :) Sorry to hear about all the infections you have been through since `Christmas` and glad to hear you are feeling better now 💪

Thanks appreciate the heads up 😄
